radioactivedecay/inventory.py

```diff
@@ -331,6 +335,46 @@ def activities(self, units: str = "Bq") -> Dict[str, float]:
         }
 
         return activities
+    
+    def energies(self, units="ev") -> Dict[str, Dict[str, float]]:
+        """
+        Returns a dictionary containing the energy released of each nuclide within the inventory.
+        Energy is sorted by decay type
+
+        Parameters
+        ----------
+        units : str, optional
+            energy units for output, ev (electron volt) or wh (watt hour)
+            Deafult is 'ev'.
+
+        Examples
+        --------
+        >>> rd.Inventory({'H-3': 1}).energies()
+        {'H-3': {'B-': 18.59202}}
```

Comment:
It looks like the output is in keV rather than eV (as the default Q_decay unit on the [live charts ](https://www-nds.iaea.org/relnsd/vcharthtml/VChartHTML.html#dcy2)site is keV)

so this means (correcting the `EV_TO_WH` constant) the default output when units="wh" is actually kWh